api/src/index.ts

```diff
@@ -1,15 +1,15 @@
-import express, { Request, Response, NextFunction } from 'express';
+import express, { Request, Response, NextFunction } from "express";
 
-const app  = express();
+const app = express();
 const port = 3000;
 
 /**
  * @todo Set using environment variables for security hardening.
  */
-app.use(function(req, res, next) {
-  res.setHeader('Access-Control-Allow-Origin',  '*')
-  res.setHeader('Access-Control-Allow-Methods', '*')
-  res.setHeader('Access-Control-Allow-Headers', '*')
+app.use(function (req, res, next) {
+  res.setHeader("Access-Control-Allow-Origin", "*");
```

Comment:
I think that these formatting changes appeared in another PR, possibly a merge conflict coming?

Reply:
Good memory. Seems like we'll find out soon!